stdlib/universe/universe.flux

```diff
@@ -8,21 +14,229 @@ import "strings"
 import "regexp"
 import "experimental/table"
 
-// now is a function option whose default behaviour is to return the current system time
+// now is a function option that, by default, returns the current system time.
+//
+// #### now() vs system.time()
+// `now()` returns the current system time (UTC). `now()` is cached at runtime,
+// so all executions of `now()` in a Flux script return the same time value.
+// `system.time()` returns the system time (UTC) at which `system.time()` is executed.
+// Each instance of `system.time()` in a Flux script returns a unique value.
+//
+// ## Examples
+//
+// ### Use the current UTC time as a query boundary
+// ```no_run
+// data
+//     |> range(start: -10h, stop: now)
+// ```
+//
+// ### Define a custom now time
+// ```no_run
+// option now = () => 2022-01-01T00:00:00Z
+// ```
+//
+// introduced: 0.7.0
+// tags: date/time
+//
 option now = system.time
 
-// Transformation functions
+// chandeMomentumOscillator applies the technical momentum indicator developed
+// by Tushar Chande to input data.
+//
+// The Chande Momentum Oscillator (CMO) indicator calculates the difference
+// between the sum of all recent data points with values greater than the median
+// value of the data set and the sum of all recent data points with values lower
+// than the median value of the data set, then divides the result by the sum of
+// all data movement over a given time period.
+// It then multiplies the result by 100 and returns a value between -100 and +100.
```

Comment:
Let's split this description into multiple sentences. One approach:

```suggestion
// The Chande Momentum Oscillator (CMO) indicator 1) calculates the difference
// between the sum of all recent data points with values greater than the median
// value of the data set, 2) calculates the sum of all recent data points with values lower
// than the median value of the data set, and then 3) divides the result by the sum of
// all data movement over a given time period.
// It then multiplies the result by 100 and returns a value between -100 and +100.
```

What is "the result" in number 3 above?